content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -667,32 +667,33 @@ For get and list, the semantics of resource version are:
 
 **Get:**
 
-| resourceVersion unset | resourceVersion="0" | resourceVersion="{non-zero version}" |
-|-----------------------|---------------------|--------------------------------------|
-| Most Recent           | Any                 | Not older than                       |
+| resourceVersion unset | resourceVersion="0" | resourceVersion="{value other than 0}" |
+|-----------------------|---------------------|----------------------------------------|
+| Most Recent           | Any                 | Not older than                         |
 
 **List:**
 
-| paging    | resourceVersion unset | resourceVersion="0" | resourceVersion="{non-zero version}" |
-|-----------|-----------------------|---------------------|--------------------------------------|
-| no limit  | Most Recent           | Any                 | Not older than                       |
-| limit="n" | Most Recent           | Any                 | Exact                                |
-
+| paging                        | resourceVersion unset | resourceVersion="0" | resourceVersion="{value other than 0}" |
+|-------------------------------|-----------------------|---------------------|----------------------------------------|
+| limit unset                   | Most Recent           | Any                 | Not older than                         |
+| limit="n", continue unset     | Most Recent           | Any                 | Exact                                  |
+| limit="n", continue="<token>" | Continue Token        | Continue Token      | Continue Token                         |
```

Comment:
* resourceVersion unset: clarify this is "Continue token, exact"?
* resourceVersion="0": I thought this was disallowed / would error
* resourceVersion="{value other than 0}": I thought this was disallowed / would error

Reply:
- `resourceVersion unset`: Fixed, thanks!
- `resourceVersion="0"`: I didn't expect this- https://gist.github.com/jpbetz/0894364d5921179ea24b2cb8ea7aaf8c
- `resourceVersion="{value other than 0}"`: 400 Bad Request -  https://gist.github.com/jpbetz/82a712788b0a0a7006b4217ee498d680

I'll document

Reply:
For `resourceVersion="0"`, I've documented this with "Invalid, but treated as Continue Token, Exact". I'm not happy about that case :(